### PR TITLE
boards: changing the property name and missed properties

### PR DESCRIPTION
**Pull request by yerabolu**
Updated the dts and overlay files to reflect the changes in the property names and added the missing ones.

### CODE REVIEW

**Review by albertofloyd — APPROVED** [review]

**Review by VenkatKotakonda — APPROVED** [review]

File: out_of_tree_boards/boards/arm/mec1501_mtl_p/mec1501_mtl_p.dts [hunk 1]
```diff
@@ -134,6 +134,10 @@
 };
 
 &spi0 {
+	status = "okay";
+	port_sel = <0>;
```
Comment (albertofloyd) [inline out_of_tree_boards/boards/arm/mec1501_mtl_p/mec1501_mtl_p.dts:138]: shouldn't this be port-sel & chip-select (with dash)

File: out_of_tree_boards/boards/arm/mec172x_mtl_s/mec172x_mtl_s.dts [hunk 2]
```diff
@@ -188,10 +188,10 @@
 
 &spi0 {
 	status = "okay";
+	compatible = "microchip,xec-qmspi-ldma";
 	clock-frequency = <4000000>;
 	lines = <4>;
 	chip_select = <0>;
```
Comment (albertofloyd) [inline out_of_tree_boards/boards/arm/mec172x_mtl_s/mec172x_mtl_s.dts:194]: chip-select

File: out_of_tree_boards/boards/arm/mec172x_mtl_s/mec172x_mtl_s.dts [hunk 3]
```diff
@@ -188,10 +188,10 @@
 
 &spi0 {
 	status = "okay";
+	compatible = "microchip,xec-qmspi-ldma";
```
Comment (albertofloyd) [inline out_of_tree_boards/boards/arm/mec172x_mtl_s/mec172x_mtl_s.dts:191]: This is missing for mec172xmodular overlay

Comment (albertofloyd) [reply out_of_tree_boards/boards/arm/mec172x_mtl_s/mec172x_mtl_s.dts:191]: Though this should have been there in zephyr/boards/arm/mec172modular/ maybe need to submit a patch into mainline

**Pull request closed without merging**